core/domain/exp_jobs_one_off.py

```diff
@@ -478,6 +481,93 @@ def reduce(key, values):
         yield (key, output_values)
 
 
+class PopulateXmlnsAttributeInExplorationMathSvgImagesJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """One-off job to populate xmlns attribute in the math-expression svg
+    images.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        fs = fs_domain.AbstractFileSystem(fs_domain.GcsFileSystem(
+            feconf.ENTITY_TYPE_EXPLORATION, item.id))
+        filepaths = fs.listdir('image')
+        image_count = 0
+        for filepath in filepaths:
+            filename = filepath.split('/')[-1]
+            if not re.match(constants.MATH_SVG_FILENAME_REGEX, filename):
+                continue
+            old_svg_image = fs.get(filepath)
+            new_svg_image = (
+                html_validation_service.get_svg_with_xmlns_attribute(
+                    old_svg_image))
+            try:
+                image_validation_services.validate_image_and_filename(
+                    new_svg_image, filename)
+            except Exception as e:
+                yield (
+                    'FAILED validation',
+                    'Exploration with id %s failed image validation for the '
+                    'filename %s with following error: %s' % (
+                        item.id, filename, e))
+            else:
+                fs.commit(
+                    filepath.encode('utf-8'), new_svg_image,
+                    mimetype='image/svg+xml')
+                image_count += 1
+        yield ('SUCCESS', image_count)
```

Comment:
Please modify this to yield 'UNCHANGED' if the image wasn't changed, and 'SUCCESS - CHANGED' if the image was changed. For the former case, provide just the count; for the latter case, provide details about the filename. Then this job can be compared with the results of the audit job.

Reply:
Done.